src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,89 @@
-import { useState } from 'react';
+import { FC, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie: FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
Would be simpler to have one single state for the entire form
```suggestion
  const [formData, setFormData] = useState(initialFormData);
```